stl/inc/cmath

```diff
@@ -12,9 +12,15 @@
 #include <cstdlib>
 #include <xtr1common>
 
-#if !defined(_M_CEE) && !defined(__clang__)
+#if !defined(_M_CEE) && !defined(__clang__) && !defined(__CUDACC__)

```

Comment:
`&& !defined(__INTEL_COMPILER)`? @erichkeane, do I understand correctly that (1) `__INTEL_COMPILER` identifies the "classic" Intel C++ compiler and (2) the Intel LLVM-based compilers define `__clang__`?

Reply:
Yes, that is correct.  The Intel LLVM-based compilers treat themselves as just clang compilers when we can.  Our SYCL compiler (at https://github.com/intel/llvm) has some additional macros for SYCL mode, and our closed-source downstream has some other detection as well, but __clang__ would be enough to exclude our llvm-based compilers from this (but obviously not classic ICC).

Reply:
Thank you for the confirmation; I'll make this a definite suggestion then:
```suggestion
#if !defined(_M_CEE) && !defined(__clang__) && !defined(__CUDACC__) && !defined(__INTEL_COMPILER)
```
(Aside: we should consider having C1XX predefine `_IS_ACTUALLY_MSVC` or something so we don't have to write conditionals that exhaustively exclude everything else.)

Reply:
At least, not until other MSVC-emulating compilers start defining it too 🙃